tests/snap/backticks.test.scala

```diff
@@ -0,0 +1,14 @@
+
+object `Backtics test` {
+    val x = MediaRange.`*/*`
+ 
+    val y = `MediaRange`.`*/*`
```

Comment:
Looking forward to enjoy this fix on GitHub.com 😄 

<img width="355" alt="Screenshot 2019-06-25 at 13 26 48" src="https://user-images.githubusercontent.com/1408093/60096264-c92f2a00-9750-11e9-9998-a2f3835aa9ff.png">